tracer/src/Datadog.Trace/RemoteConfigurationManagement/Product.cs

```diff
@@ -42,6 +42,13 @@ public void AssignConfigs(List<RemoteConfiguration> changedConfigs)
                 }
             }
 
+            Log.Debug(
+                "Received {ConfigsAmount} Remote Configuration records for product {Name}, " +
+                      "of which {FilteredAmount} matched the predicate.",
+                changedConfigs.Count.ToString(),
+                Name,
+                (filteredConfigs?.Count ?? 0).ToString());
```

Comment:
nit: no need for the `ToString()` (unnecessary allocation if debug is not enabled)
```suggestion
                (filteredConfigs?.Count ?? 0));
```

Reply:
Done in 07e24548631653b407159077cf3de015d03b5cf6